docs/registry/create/_index.md

```diff
@@ -654,11 +654,12 @@ To [add a module](/registry/configure/) to the configuration of your robot, you
 Your options for completing this step are flexible, as this file does not need to be in raw binary format.
 
 {{% tabs %}}
-{{% tab name="Python" %}}
+{{% tab name="Python: venv (recommended)" %}}
 
-One option is to create and save a new shell script (<file>.sh</file>) that runs your module at your entry point (main program) file.
+Create and save a new shell script (<file>.sh</file>) that runs your module at your entry point (main program) file.
 
-Make sure to set up a Python virtual environment in the directory your module is in to ensure your module has access to any required libraries:
+Make sure to [prepare a Python virtual environment](/program/python-venv/) in the directory your module is in to ensure your module has access to any required libraries.
+Compile your module as follows:
```

Comment:
this does not compile anything it just adds a bash file that installs the dependencies and runs the python file
```suggestion
Make sure to [prepare a Python virtual environment](/program/python-venv/) in the directory your module is in to ensure your module has access to any required libraries:
```

Reply:
can you actually also change the header for this to "Prepare the module for execution"

Reply:
Done! And oops, did not realize